Add instructions to test against another repo's branch.

carlosalberto opened the pull request.

carlosalberto committed:
Add instructions to test against another repo's branch.

malafeev commented on README.md:
```diff
@@ -6,3 +6,38 @@ tester examples of common instrumentation patterns
+
+By default the version for a build looks like `0.30.1-SNAPSHOT`, which helps to keep the stable and the testing one side by side.
+
+Next, look for the `<properties>` section in the `pom.xml` file under this directory, updating the `opentracing` version; and also add a dependency to the `usecases` package:
```
where `usecases` package comes from? I don't see any `artifactId` with `opentracing-usecases`

carlosalberto replied:
Damn, that slipped from the `Scope` PR. Will fix in a bit.

malafeev commented on README.md:
```diff
@@ -6,3 +6,38 @@ tester examples of common instrumentation patterns
+In order to test these examples against an `opentracing-java` branch, you have to install it in local Maven repository:
+
+    $ cd opentracing_java/
+    $ git checkout your-testing-branch
```
how about to rename `your-testing-branch` to  `your-opentracing-java-branch` ?

malafeev commented on README.md:
```diff
@@ -6,3 +6,38 @@ tester examples of common instrumentation patterns
+
+By default the version for a build looks like `0.30.1-SNAPSHOT`, which helps to keep the stable and the testing one side by side.
+
+Next, look for the `<properties>` section in the `pom.xml` file under this directory, updating the `opentracing` version; and also add a dependency to the `usecases` package:
```
here `pom.xml` of `opentracing-java` or `java-examples`? I'm confused :)

malafeev commented on README.md:
```diff
@@ -6,3 +6,38 @@ tester examples of common instrumentation patterns
+
+    $ mvn test
+
+Any further changes under this repository will not need re-installing anything, simply re-running this last command.
```
which repository? where I am? :)

malafeev replied:
My concern that it's not very clear what to do. I was confused several times during reading.
Could you add more `stupid` steps (explanations, clarifications...)? Imagine user see it first time.

malafeev commented:
My concern that it's not very clear what to do. I was confused several times during reading.
Could you add more stupid steps (explanations, clarifications...)? Imagine user see it first time.

carlosalberto committed:
Improve the README.

Remove the `usecases` part (which belongs to a specific PR),
and to make clear what pom.xml to edit and where to run mvn.

carlosalberto commented:
Hey! Thanks for the feedback. Added a few small clarifications here and there ;)
